ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -199,8 +199,13 @@ open class BadgeView: UIView {
             if let customDisabledLabelTextColor = _disabledLabelTextColor {
                 return customDisabledLabelTextColor
             }
+
             let textDisabledColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foregroundDisabled1])
-            return style == .default ? textDisabledColor : (isSelected ? self.selectedLabelTextColor : self.labelTextColor)
+            if style == .default {
+                return UIColor(light: UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.brandForegroundDisabled1]), dark: textDisabledColor)
```

Comment:
Since this is still internal, you could initialize a dynamic color with the `ColorValue`s you want instead of making three `UIColor`s. Something like:
```
let colors = fluentTheme.aliasTokens.colors
let textDisabledColor = colors[.foregroundDisabled1]
if style == .default {
  return UIColor(dynamicColor: DynamicColor(colors[.brandForegroundDisabled1].light, dark: textDisabledColor.dark))
} else {
  return UIColor(dynamicColor: textDisabledColor)
}
```

Reply:
i thought about it. but i didn't like the multiple lets that seemed fragmented. ultimately this entire class needs to be tokenize which isn't yet. I think once we have the tokenset define it will be much cleaner.